vlib/v/gen/native/tests/expressions.vv

```diff
@@ -1,33 +1,9 @@
 fn print_number(n int) {
-	if n == 0 {
-		println('0')
-	}
-	if n == 1 {
-		println('1')
-	}
-	if n == 2 {
-		println('2')
-	}
-	if n == 3 {
-		println('3')
-	}
-	if n == 4 {
-		println('4')
-	}
-	if n == 5 {
-		println('5')
-	}
-	if n == 6 {
-		println('6')
-	}
-	if n == 7 {
-		println('7')
-	}
-	if n == 8 {
-		println('8')
-	}
-	if n == 9 {
-		println('9')
+	for i := 0; n == i; i++ {
+		println('$n')
+		if n >= 10 && i >= 10 {
+			break
+		}
```

Comment:
This is here to test the native backend. It currently can not handle loops. Try for example:
`./v -b native vlib/v/gen/native/tests/expressions.vv`

Before this change: 
```
call fn (main.print_number)
call fn (main.print_number)
call fn (main.print_number)
call fn (main.print_number)
call fn (main.test_add)
start
test_add()
2
3
5
6
end
```
... but after: 
```
native.stmt(): bad node: v.ast.ForCStmt
call fn (main.print_number)
call fn (main.print_number)
call fn (main.print_number)
call fn (main.print_number)
call fn (main.test_add)
start
test_add()
end
```